src/models/Scholarships.ts

```diff
@@ -155,6 +156,11 @@ class Scholarships extends FirestoreCollection<ScholarshipData> {
                 data.requirements?.grades,
                 opts.grades
               ) &&
+              // major filter
+              requirementMatchesFilter(
+                data.requirements?.majors,
+                opts.majors
```

Comment:
Something that just came to mind is how we match up free form values when case mismatches. Maybe we should transform each of these lists to lower case so that case doesn't matter? I think you need something like `.map(s => s.toLowerCase())`.

Reply:
Also I'm realizing it's going to be hard to actually make these grade level and majors filters work when we don't have that data inputted yet for all scholarships. Something we should probably follow up on (I left a comment on Slack).